tsconfig.json

```diff
@@ -0,0 +1,109 @@
+{
+  "compilerOptions": {
```

Comment:
Can we remove the commented out stuff?

Reply:
This is a generated file (from the npm typescript setup) that shows you the options available. I kind of think of it as showing these options are currently off, but we could turn them on in the future. Could remove them if we think they're confusing or leave them if we want easy access to those options.

Reply:
Gotcha, I feel like they can still be removed though. Can always look at the npm typescript setup later on if we need to add stuff.